recipes/ckanweb-deploy.rb

```diff
@@ -124,7 +149,7 @@
 # Create NGINX Config files
 #
 
-node.override['datashades']['app']['locations'] = "location ~ ^#{node['datashades']['ckan_web']['endpoint']} { proxy_pass http://localhost:8000; proxy_set_header Host $host; proxy_set_header X-Real-IP $remote_addr; proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for; }"
+node.override['datashades']['app']['locations'] = "location ~ ^#{node['datashades']['ckan_web']['endpoint']} { proxy_pass http://localhost:8080; proxy_set_header Host $host; proxy_set_header X-Real-IP $remote_addr; proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for; }"
```

Comment:
so port 8000 to 8080

Reply:
uWSGI will run on port 8080. Apache is still on 8000 for now but unused, will be cleaned up later.